sig-security/security-audit-2021/RFP.md

```diff
@@ -63,9 +63,12 @@ If your proposal includes subcontractors, please include relevant details from t
 
 ### Anticipated Selection Schedule
 
-This RFP will be open between 2021/02/08 and 2021/04/09.
+This RFP will be open until 4 proposals have been received.
+The RFP closing date will be set 1 calendar week after the fourth proposal is received.
```

Comment:
Maybe 2 weeks, as the sig meets every other week?

Reply:
Updated RFP with suggestion